web/src/CommandPalette.jsx

```
@@ -24,18 +24,68 @@ export default function CommandPalette({ isOpen, onClose, setActivePanel, action
+            icon: '👤',
+            desc: 'Open in Contacts',
+            action: (setActivePanel) => {
+              setActivePanel('contacts');
+              // We can hack search param into DOM or state if accessible
+              // For now just open contacts
+            }
+          });
+        }
+      };
+
+      trustedContacts.forEach(c => addContact(c, 'trusted'));
```

Biome (2.3.13)

[error] This callback passed to forEach() iterable method should not return a value.

Either remove this return or remove the returned value.


(lint/suspicious/useIterableCallbackReturn)